src/dashboard/navigation/AgentList.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import ListItem from "@mui/material/ListItem";
+import ListItemButton from "@mui/material/ListItemButton";
+import ListItemText from "@mui/material/ListItemText";
+import List from "@mui/material/List";
+import { Link } from "react-router-dom";
+
+export default function AgentList() {
+  const Agent = [
+    { id: "Bucharati", text: "Agent Bucharati" },
+    { id: "Mister", text: "Agent Mista" },
+    { id: "Dio", text: "Agent Dio" },
+    { id: "Jojo", text: "Agent Jojo" },
+  ];
+
+  return (
+    <List>
+      <ListItem>
+        <ListItemButton button component={Link} to="/agentbucharati">
```

Comment:
After you update your router which leverage react router url parameter, please update the link here to be like `/agent/bucharati`
`/agent/Mister`

Hint: The `<ListItem>` can leverage `.map` function for render. For example: `Agent.map((content) => { return <ListItem ...})`